src/baldr/complexrestriction.cc

```diff
@@ -0,0 +1,40 @@
+#include "baldr/complexrestriction.h"
+
+namespace valhalla {
+namespace baldr {
+
+bool CheckPatchPathForRestrictions(
```

Comment:
might be better in a generic thor header that has something to do with pathfinding like astar.h

Reply:
:heavy_check_mark: